app/controllers/merchant_items_controller.rb

```diff
@@ -2,4 +2,9 @@ class MerchantItemsController < ApplicationController
   def index
     @merchant = Merchant.find(params[:merchant_id])
   end
-end
+
+  def show
+    @merchant = Merchant.find(params[:merchant_id])
+    @item = @merchant.items.find(params[:item_id])
```

Comment:
This is again one of those things that could be moved to the view for "skinny controller"

Reply:
I don't know why I'm so scared to do that.  I'm going to make these changes and be bolder with my view files moving forward.

Reply:
Well we were originally taught the first way and then later taught the second. I struggle with that as well. I still have a bad habit of not using rails forms in favor of pure HTML...